tests/unit/test_coco_format.py

```diff
@@ -1864,6 +1864,99 @@ def test_can_save_and_load_keypoints(self):
                 target_dataset=target_dataset,
             )
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_keypoints_in_skeleton_order_when_some_points_absent(self):
+        source_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id=1,
+                    subset="train",
+                    media=Image.from_numpy(data=np.zeros((5, 5, 3))),
+                    annotations=[
+                        Skeleton(
+                            [
+                                Points([0, 2], [1], label=2),
```

Comment:
```suggestion
                                Points([0, 2], [Points.Visibility.occluded], label=2),
```
Please use enum values

Reply:
done